eng/emitter-package.json

```diff
@@ -18,6 +18,7 @@
     "@azure-tools/typespec-autorest": "~0.61.0",
```

Comment:
Please "git merge origin/main" and revert all changes to this file (if any). It should not appear in your PR